app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java

```diff
@@ -0,0 +1,91 @@
+package net.gsantner.markor.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.ScrollView;
+
+public class DraggableScrollbarScrollView extends ScrollView {
+
+    private boolean _isFastScrolling = false;
+    private boolean _fastScrollEnabled = true;
+    private boolean _ltr = true;
+    private int _thumbHeight;
+
+    public DraggableScrollbarScrollView(Context context) {
+        super(context);
+    }
+
+    public DraggableScrollbarScrollView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public DraggableScrollbarScrollView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
```

Comment:
so custom steps are not required anymore at constructor, do I see right?

In that case it should be possible to remove all constructors...and super's contructors (`Scrollview()`) are automatically called